webpack/components/extensions/HostDetails/Tabs/ErrataTab.scss

```diff
@@ -8,4 +8,9 @@
   margin: 0 20px;
   margin-bottom: 2em;
   margin-top: 2em;
+}
+
+
+.pf-c-toggle-group__text {
+  color: black;
```

Comment:
Super weird that you had to do this!  I wonder why :thinking:

Reply:
Try removing it and see the picture :)